drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -476,6 +495,13 @@ static int bt_ipm_open(void)
 {
 	int err;
 
+	/* Start RX thread */
+	k_thread_create(&ipm_rx_thread_data, ipm_rx_stack,
+			K_THREAD_STACK_SIZEOF(ipm_rx_stack),
+			(k_thread_entry_t)bt_ipm_rx_thread, NULL, NULL, NULL,
+			K_PRIO_COOP(CONFIG_BT_RX_PRIO - 1),
```

Comment:
I think this PR makes your thread the RX prio thread. So should use CONFIG_BT_RX_PRIO here.

Reply:
I can change, but why would it be different from SPI or USERCHAN ?

Reply:
Forget what I said. I thought `CONFIG_BT_RX_PRIO` was the priority if the RX prio thread, but it is the RX thread. sorry for the confusion.